panther-labs/pounce
# Feat Dropdown

### Background

We need a component that will act as a menu popup, i.e. a dropdown that will appear as soon as a button is clicked. This component is fully accessible and is triggered through an element click

### Changes

- Adds Dropdown component

### Testing
It should:

- open when its `trigger` is clicked
- close when a menu option is clicked
- close when the background is clicked
- allow keyboard navigation
